docs/readmes/orc8r/dev_minikube.md

```diff
@@ -122,8 +122,22 @@ Apply the secrets
 cd ${MAGMA_ROOT}/orc8r/cloud/helm/orc8r
 helm template orc8r charts/secrets \
   --namespace orc8r \
+  --set-string secret.certs.enabled=true \
+  --set-file 'secret.certs.files.rootCA\.pem'=${CERTS_DIR}/rootCA.pem \
+  --set-file 'secret.certs.files.bootstrapper\.key'=${CERTS_DIR}/bootstrapper.key \
+  --set-file 'secret.certs.files.controller\.crt'=${CERTS_DIR}/controller.crt \
+  --set-file 'secret.certs.files.controller\.key'=${CERTS_DIR}/controller.key \
+  --set-file 'secret.certs.files.admin_operator\.pem'=${CERTS_DIR}/admin_operator.pem \
+  --set-file 'secret.certs.files.admin_operator\.key\.pem'=${CERTS_DIR}/admin_operator.key.pem \
+  --set-file 'secret.certs.files.certifier\.pem'=${CERTS_DIR}/certifier.pem \
+  --set-file 'secret.certs.files.certifier\.key'=${CERTS_DIR}/certifier.key \
+  --set-file 'secret.certs.files.nms_nginx\.pem'=${CERTS_DIR}/controller.crt \
+  --set-file 'secret.certs.files.nms_nginx\.key\.pem'=${CERTS_DIR}/controller.key \
   --set-file 'secret.certs.files.fluentd\.pem'=${CERTS_DIR}/fluentd.pem \
-  --set-file 'secret.certs.files.fluentd\.key'=${CERTS_DIR}/fluentd.key |
+  --set-file 'secret.certs.files.fluentd\.key'=${CERTS_DIR}/fluentd.key \
+  --set=docker.registry=${IMAGE_REGISTRY_URL} \
+  --set=docker.username=${IMAGE_REGISTRY_USERNAME} \
+  --set=docker.password=${IMAGE_REGISTRY_PASSWORD} |
```

Comment:
All secrets need to be applied together. Otherwise pods fail because they cannot find relevant certs.